lib/tasks/dev.rake

```diff
@@ -21,7 +21,11 @@ namespace :dev do
       first_name: 'Some',
       last_name: 'One'
     )
-    profile.encrypt_pii(loa3_user.user_access_key, pii)
+    recovery_code = profile.encrypt_pii(loa3_user.user_access_key, pii)
     profile.activate
+
+    Kernel.puts "===="
```

Comment:
We've never needed the recovery code that we generate for this user until now, so it's new functionality